src/api.raml

```diff
@@ -497,6 +497,13 @@ traits:
         is:
           - paged: { resources: Collections }
           - subjected: { resources: Collections }
+        queryParameters:
+            containing:
+                description: |
+                    Only return collections containing at least one of these items.
+                type: string[]
+                required: false
+                example: [article/1234, interview/5678]
```

Comment:
I've always been suspicious of pushing unrelated changes through for the sake of consistency in an awkward and inconsistent bit of the program. It strikes me that what collections needs here (a list of id and type pairs) feels awkward because it's not in keeping with the rest of the api, and that there are sort-of similar cases where a type is supplied along with an id in the path, like `/article/09560/page-views`. And you're suggesting to make it `/page-views?foo=article:09560`, right? or is it `/article:09560/page-views` ?

Either way, it seems like you're trying to backport an awkward design decision in one app in one case to multiple existing apps and future apps for all cases, just for consistency's sake. This ignores that metrics doesn't support multiple types and ids in one request.

`/collections?containing=article:09560&containing=interview:12345` seems fine to me. As for delimiters, I prefer the pipe `|` but I really don't care in this case. 

When we have two such instances of where multiple types and ids must be passed, we could look at it again then and see how well collections fared using whatever decision you make now.

:-1: